benchmarks/NonlinearProblem/nonlinear_solver_23_tests.jmd

```diff
@@ -25,15 +25,13 @@ solvers = [ Dict(:alg=>NewtonRaphson()),
             Dict(:alg=>LevenbergMarquardt()),
             Dict(:alg=>CMINPACK(method=:hybr)),
             Dict(:alg=>CMINPACK(method=:lm)),
-            Dict(:alg=>NLSolveJL()),
-            Dict(:alg=>KINSOL())]
+            Dict(:alg=>NLSolveJL())]
```

Comment:
Let's add NewtonRaphson and Anderson from NLSolveJL to this list?

Reply:
@TorkelE these two were left off, can you make a separate PR to add them?

Reply:
Yes, will go through which solvers to run again with Avik tomorrow